light_curves/code_src/sample_selection.py

```diff
@@ -374,15 +404,62 @@ def get_paper_sample(coords, labels, *, paper_link="2019A&A...627A..33D", label=
         print("number of sources added from "+str(label)+" :"+str(len(paper_coords)))
 
 
-def clean_sample(coords_list, labels_list, verbose=1):
-    """Makes a unique sample of skycoords and labels with no repeats. Attaches an object ID to the coords.
+def get_papers_list_sample(coords, labels, *, paper_kwargs=[dict(),]):
```

Comment:
@troyraen I thought you once suggested that I not write 1 or 2 liner functions.  I don't care at all here, but am trying to learn from you.  So what is your logic in making this a function?

Reply:
This makes it a lot easier for #215 to gather samples from more than one paper. Without this, `get_paper_sample` would require special handling in #215 because the code would have to check whether the user wanted more than one paper, and then call `get_paper_sample` multiple times. It's fewer lines of code to add this function than it would be to do the special handling.

(Shooby had asked me for a sample including multiple papers, so I had to think about the best way to do this. Otherwise, I probably wouldn't have noticed or spent any time on it.)

Reply:
But, there's no great reason for `get_papers_list_sample` and `get_paper_sample` to be two different functions. There, I was just choosing the option that was least intrusive to the existing code. But they are both pretty simple and could be combined, if you want?

Reply:
No need to combine the functions